index.html

```diff
@@ -196,14 +196,16 @@ <h2><dfn>Performance Timeline</dfn></h2>
       <li>a <dfn>performance observer task queued flag</dfn></li>
       <li>a <dfn>list of <a>registered performance observer</a> objects</dfn>
       that is initially empty</li>
-      <li>a <dfn>performance entry buffer</dfn> to store
-      <a>PerformanceEntry</a> objects that is initially empty
+      <li>a <dfn>performance entry buffer</dfn> per observed
```

Comment:
What does 'observerd entryType' mean? Maybe the correct way to do this would be to say that other specs that define new entry types need to specify a new buffer for that type. And here we have a map entryType->buffer.

Reply:
Yeah observed didn't belong here.  

I updated to map.  Not clear what you mean by "other specs that define new entry types need to specify a new buffer for that type".  This map will contain buffers for all entryTypes, we don't obtain those buffers elsewhere.

Reply:
I updated this map again after we spoke offline.  PTAL

Reply:
Agree with @npm1 here. We'd want each spec to explicitly "register" its buffer, in a similar way to the way each spec is [registering supported entry types](https://w3c.github.io/performance-timeline//#dfn-register-a-performance-entry-type). We need that registration to include the full tuple, so indicate a) its entryType b) its max size c) its visibility from the timeline

Reply:
Is this something I need to explicitly call out somewhere that each spec needs to do this?

Reply:
You need to add an algorithm that performs that registration here for a generic entry type, and each spec would then need to call into that algorithm to register its buffer.